app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::CommentsController < Api::ApplicationController
+  def index
```

Comment:
- [OPTIONAL ] You're setting the comment's `author` and `post` attributes separately. However, you may want to consider using `Rails' built-in` associations to automatically set these attributes for you. For example, if you set up the Comment model with `belongs_to :author`, `class_name: User` and `belongs_to :post`, you can then create a new comment with `@post.comments.create(comment_params.merge(author: current_user))`, which will automatically set the post and author attributes correctly. :wink: